### PR TITLE
add kernel 5.19-rt10

### PR DESCRIPTION
**Pull request by fps**
Sometimes you just need a newer kernel.. ;) Still untested (my machine is slooow)

### CODE REVIEW

File: pkgs/os-specific/linux/kernel/update.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ set -o pipefail
 
 BRANCHES=(
   "4.4" "4.9" "4.14" "4.18" "4.19"
-  "5.0" "5.4" "5.6" "5.9"
+  "5.0" "5.4" "5.6" "5.9" "5.19"
```
Comment (sorki) [inline pkgs/os-specific/linux/kernel/update.sh:9]: Looks like this should contain `5.16` and `5.17` as well.

**Comment by magnetophon** [comment]
@fps Thanks! Builds and runs great for me.
If you could you add the changes @sorki suggested, I'll merge.

**Comment by magnetophon** [comment]
@fps Thanks!

I'm sorry, but I only just noticed that your PR contains multiple kernels, and when I merge it, those will all be under one commit message, iiuc?
You might want to split this into 3 PR's.

@henrytill Any thoughts on this?

**Comment by fps** [comment]
Since I haven't squashed the commits yet the individual commits should be kept, no? I can rewrite the history though to make the commit messages more uniform..

**Comment by magnetophon** [comment]
> Since I haven't squashed the commits yet the individual commits should be kept, no?

I wasn't sure, but I think you are right.

> I can rewrite the history though to make the commit messages more uniform..

That would be great.
So far, most commits that add a kernel have been in the form: ``kernel: add 4.18.5-rt3`` so maybe go with that.

Thanks!

Have you gotten around to testing the various kernels?

**Comment by henrytill** [comment]
> @fps Thanks!
> 
> I'm sorry, but I only just noticed that your PR contains multiple kernels, and when I merge it, those will all be under one commit message, iiuc? You might want to split this into 3 PR's.
> 
> @henrytill Any thoughts on this?

Commits all look good to me, no need to modify history on my accord.  Our default rebase merge will also preserve history as well.

@fps: greatly appreciate your work here!

**Comment by magnetophon** [comment]
@henrytill So I guess we wait for some positive test reports on all kernels and then we can merge?
Personally I have only tested 5.19-rt10 so far.

**Comment by henrytill** [comment]
> @henrytill So I guess we wait for some positive test reports on all kernels and then we can merge? Personally I have only tested 5.19-rt10 so far.

I dusted off my [test script](https://github.com/musnix/musnix/blob/master/tests/default.nix) and am currently running against `5_16_rt` and `5_17_rt`.  Will report back.

**Comment by henrytill** [comment]
That's good enough for me!

**Comment by fps** [comment]
Hi again!

Sorry, I'm not too quick to respond. Having a toddler does that ;) Anyways, yeah, I booted into all of these kernels and ran them for a while. Except for a regression I noticed and reported here they worked fine: https://lore.kernel.org/linux-rt-users/ff61ad6d-a8d3-e83e-1296-bf7d1f503aeb@gmx.net/T/#t

The culprit was power management so I managed to work around it on all of these kernels.

**Pull request merged**